quodlibet/tests/test_util_collection.py

```diff
@@ -619,3 +636,37 @@ def test_masked_handling(self):
             self.failUnless(song in pl)
 
             lib.destroy()
+
+
+class TestXPSFBackedPlaylist(TFileBackedPlaylist):
+    Playlist = XSPFBackedPlaylist
+
+    def setUp(self):
+        super().setUp()
+        self.temp = mkdtemp()
+        self.temp2 = mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.temp)
+        shutil.rmtree(self.temp2)
+
+    def test_from_songs(self):
+        pl = XSPFBackedPlaylist.from_songs(self.temp, NUMERIC_SONGS)
```

Comment:
Shouldn't this also use `self.Playlist` like in `TFileBackedPlaylist`

Reply:
Good point, in fact even better not sure why that method's still there as the base version now handles it